app/views/invitations/_invitation.html.erb

```diff
@@ -1,29 +1,29 @@
 <dl>
 	<dt>Sender</dt>
-    <dd><%= creator_link(invitation) %></dd>
+    <dd><%= creator_link(@invitation) %></dd>
```

Comment:
I needed to make changes here to fix issues with how `@invitation` is referenced (`locals` versus just passing it in). For everyone's sanity, I'll do the rest of the erb lint fixes in https://github.com/otwcode/otwarchive/pull/5000 since that has most of them already anyways